tests/patterns/python/wildcard_qualified_mvar.sgrep

```diff
@@ -0,0 +1 @@
+$A.x
```

Comment:
Same here, I don't remember SR asking for this? And what happens here if we have multiple wildcard imports? (We could think of some corner cases involving multiple wildcard imports.)